### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by tijmenb**
👉[Rendered version](https://github.com/alphagov/govuk-cdn-config/blob/readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,55 @@
 # GOV.UK CDN config
 
-Configuration for GOV.UK's content delivery network (CDN).
+Configuration for GOV.UK's content delivery network (CDN). You can read more about [how GOV.UK CDN works](https://docs.publishing.service.gov.uk/manual/cdn.html) in the developer docs.
 
-Configures the [Fastly CDN](https://fastly.com) from version-controllable VCL and
-YAML files, using Fastly's [API](https://docs.fastly.com/api/).
+⚠️ This repo has some [documented tech debt](https://trello.com/c/y6MIgxjp). It doesn't conform to GOV.UK standards, and lacks sufficient testing. Approach code changes with caution.
```
Comment (timblair) [inline README.md]: Is this a bit alarmist? There's a fine line between caution vs. putting people off wanting to do something...

File: README.md [hunk 2]
```diff
@@ -1,29 +1,55 @@
...
+
+Invoked via the [CDN: deploy service](https://deploy.publishing.service.gov.uk/job/Deploy_CDN) Jenkins job.
+
+This script allows you to configure a number of Fastly services:
+
+| service name | domain | description |
+| --- | --- | --- |
+| apt | apt.publishing.service.gov.uk | GOV.UK's Debian package repository |
+| assets | assets.publishing.service.gov.uk | the GOV.UK domain for uploads and static assets |
+| performanceplatform | service.gov.uk | the Performance Platform (https://www.gov.uk/performance) |
```
Comment (timblair) [inline README.md]: Missing the full domain(s) for the performance platform service.

Comment (tijmenb) [reply README.md]: 👍 added

File: README.md (hunk 1 above)
Comment (tijmenb) [reply README.md]: 👍 that's a great point, I've rewritten it to be explicit about what I mean by "approach code changes with caution" (which is manual testing).

**Pull request merged**